linchpin/api/__init__.py

```diff
@@ -338,7 +338,7 @@ def run_playbook(self, pinfile, targets='all', playbook='up'):
             if 'post' in self.pb_hooks:
                 self.hook_state = '{0}{1}'.format('post', playbook)
 
-        return results
+        return return_code, results
```

Comment:
As a rule, when returning, use parentheses `eg. (return_code, results)` around the items being returned when they are grouped.